packages/components/src/core/Icon/index.tsx

```diff
@@ -65,18 +64,18 @@ const Icon = forwardRef(function Icon<IconName extends keyof IconNameToSizes>(
           className={className}
           fillcontrast="white"
           viewBox="0 0 22 22"
-          component={icon()}
+          component={iconItem()}
           sdsSize={sdsSize}
           sdsType={sdsType}
-          sdsIcon={sdsIcon}
```

Comment:
On the `sdsIcon` vs `icon` I think having a more general name like `icon` works if we want to force either an sds one or a custom icon. There's probably a better name for it than `icon` though.

Reply:
Yeah good point, since `sdsIcon` could confuse users that it's expecting SDS specific icon stuff, which a custom icon wouldn't seem to fall into that category

`icon` sounds good and if we think of something better later, we can always update 😆 !